### ISSUE TITLE
Bots Can Get Stuck in Infinite Pathing Recalc Loop for Incomplete Paths

### ISSUE THREAD

**Issue by dwesterwick** [dwesterwick/SPTQuestingBots#21]
When incomplete paths are recalculated, Unity sometimes alternates between two incomplete paths, so bots never actually get "stuck". Instead, they bounce between two points forever.

**Comment by Circuitous** [comment]
May be fixed by Drakia reintroducing the FindPath patch into [Waypoints 1.7.1 ](https://github.com/DrakiaXYZ/SPT-Waypoints/releases/tag/1.7.1) on 2025-06-19

**Comment by dwesterwick** [comment]
Yeah, the Waypoints update was made to fix an issue that caused bots to teleport if they couldn't find complete paths. 

This issue is more related to bots moving back and forth between two of the same incomplete paths and not actually progressing anywhere. It's pretty rare, but it's challenging to get them to break out of the loop.
